envoy-control-core/src/test/kotlin/pl/allegro/tech/servicemesh/envoycontrol/snapshot/SnapshotUpdaterTest.kt

```diff
@@ -72,21 +88,24 @@ class SnapshotUpdaterTest {
             .hasSecuredIngressRoute("/endpoint", "client")
             .hasServiceNameRequestHeader("service")
 
-        hasSnapshot(cache, groupOf(services = listOf("existingService1")))
+        hasSnapshot(cache, groupOf(services = setOf(ServiceDependency("existingService1", settings = DependencySettings()))))
```

Comment:
is it necessary to pass argument `settings` here, which is equal to its default value?